### PR TITLE
Changes in Variables_and_datatypes

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: docs/course_materials/exercises/03_Variables_and_datatypes/class_exercise_1.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ### Let's Practice Together \#1
 
-- Create a new file **my_full_name.py** using vim editor
-- Get the user's first name and save it in a new variable **first_name**
-- Get the user's last name and save it in a new variable **last_name**
-- Print **first_name** and **last_name** using %s formatting
-- Save the file and run it on the terminal
+- Create a file *variables_class.py*
```
Comment (sleviim) [inline docs/course_materials/exercises/03_Variables_and_datatypes/class_exercise_1.md:3]: wasn't this one changed by shir already?

Comment (shiramax) [reply docs/course_materials/exercises/03_Variables_and_datatypes/class_exercise_1.md:3]: I just replaced the place of the files/

File: docs/course_materials/variables_data_types.html [hunk 2]
```diff
@@ -463,14 +472,7 @@ <h3><b>Manipulations</b></h3>
           s.strip() # -> 'hello!'
                     </code></pre>
                 </section>
-                <section>
-                    <h3><b>+ and *</b></h3>
```
Comment (sleviim) [inline docs/course_materials/variables_data_types.html:467]: why did you remove this one?
consider moving it to the extras instead.

Comment (shiramax) [reply docs/course_materials/variables_data_types.html:467]: I didn't remove it, just change the location,

**Review by sleviim — APPROVED** [review]
